ThermalTalk.QueryStatus/Program.cs

```diff
@@ -9,21 +11,30 @@ internal class Program
     {
         public static void Main(string[] args)
         {
-            const string reliancePort = "COM12";
+            var reliancePort = args.FirstOrDefault();
 
-            using (var printer = new ReliancePrinter(reliancePort))
+            Console.WriteLine($"The following portname was selected: {reliancePort}");
```

Comment:
You can test if reliancePort is null and show the error message right away instead of relying on the exception.